docs/book/how-to/trigger-pipelines/trigger-a-pipeline-from-rest-api.md

```diff
@@ -0,0 +1,27 @@
+---
+description: >-
+  Trigger a pipeline from the rest API.
+---
+
+# Trigger a pipeline from another
+
+{% hint style="info" %}
+This is a [ZenML Pro](https://zenml.io/pro) only feature. Please [sign up here](https://cloud.zenml.io) get access.
+OSS users can only trigger a pipeline by calling the pipeline function inside their runner script.
+{% endhint %}
+
+Triggering a pipeline from the REST API **only** works with pipelines that are configured with a remote stack
+(i.e. at least a remote orchestrator, artifact store, and container registry)
+
+There are three calls that need to be made in order to trigger a pipeline from the REST API:
+
+1. `GET /pipelines?name=<PIPELINE_NAME>` -> This returns the <PIPELINE_ID>
+2. `GET /pipeline_builds?pipeline_id=<PIPELINE_ID>` -> This returns the build ID
+3. `POST /pipeline_builds/<BUILD_ID>/runs` -> This runs the pipeline. You can pass the [PipelineRunConfiguration](https://sdkdocs.zenml.io/latest/core_code_docs/core-config/#zenml.config.pipeline_run_configuration.PipelineRunConfiguration) in the body
+
+The above set of REST calls means that you can only trigger a pipeline that has been [built before](../customize-docker-builds/reuse-docker-builds.md).
```

Comment:
We should probably mention again that this needs a build with a remote stack etc.